testing.go

```diff
@@ -75,7 +75,7 @@ func startAndWait(s *Server) func() {
 func deleteQueue(name string) {
 	queueURL := fmt.Sprintf("%s/queues/%s/%s", serverAPITestURL, url.PathEscape("/"), url.PathEscape(name))
 
-	req, err := http.NewRequest("DELETE", queueURL, nil)
+	req, err := http.NewRequest("DELETE", queueURL, http.NoBody)
```

Comment:
TIL!

Reply:
Yep, me too! It was a linter that complained. Same with the change from `ioutil` -> `io`/`os` :)